extra_requirements/requirements-tests.txt

```diff
@@ -43,11 +43,11 @@ pytest-cov==5.0.0
 pytest-xdist==3.6.1
 rframe==0.2.21
 scikit-learn==1.2.2
-scipy==1.13.1
-tensorflow==2.15.0.post1
+scipy==1.13.1                                     # 1.14.0 Requires-Python >=3.10
+tensorflow==2.15.0
```

Comment:
why we downgrade here?

Reply:
Fixed it (I was trying to update tensorflow and when it failed I downgraded it incorrectly)

Reply:
thanks!